src/main/java/DukeException.java

```diff
@@ -0,0 +1,5 @@
+public class DukeException extends Exception {
```

Comment:
Perhaps you can add subclasses of exceptions (ex. empty todo description, missing time), so you don't have to make a new Exception every time in Duke.java